cpu/stm32_common/periph/flashpage.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     cpu_stm32f0
+ * @ingroup     cpu_stm32_common
```

Comment:
It seems that Jenkins and Murdock agree with me, we need a compile guard. Isn't there a way to compile a periph driver only if the board provides the feature ?